src/services/user-service.js

```
@@ -0,0 +1,192 @@
+  }
+
+  static getOpenIDConfig() {
+    return (
+      getAccountDb().first(
+        `SELECT * FROM auth
+         WHERE method = ?`,
+        ['openid'],
+      ) || {}
+    );
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)